appengine/flexible/endpoints/clients/service_to_service_google_id_token/main.py

```diff
@@ -24,10 +24,9 @@
 from google.appengine.api import app_identity
 import webapp2
 
-DEFAUTL_SERVICE_ACCOUNT = "YOUR-CLIENT-PROJECT-ID@appspot.gserviceaccount.com"
+SERVICE_ACCOUNT_EMAIL = "YOUR-CLIENT-PROJECT-ID@appspot.gserviceaccount.com"
```

Comment:
I think you can grab this from `app_identity`: `app_identity. get_service_account_name()`